packages/bsp/fxblox-rk1/fxblox-rk1-usb2-init.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Init USB2 for fxblox rk1
+
+[Service]
+ExecStart=/usr/bin/sh -c "echo host > /sys/kernel/debug/usb/fc000000.usb/mode"
```

Comment:
Why not use the following config in dts?

```
usb-role-switch;
role-switch-default-mode = "host";
```

Reply:
I used this like orange pi. https://github.com/armbian/build/blob/main/config/boards/orangepi5.conf
but I can test your way.

Reply:
@efectn maybe good idea for opi5 too

Reply:
Agree that's good idea, will do it. I wonder why they made it otg as default